### PR TITLE
Fixed small review and formatting issues

### PR DESCRIPTION
**Pull request by KrzysztofCwalina**

### CODE REVIEW

**Comment by Petermarcu** [comment]
Assuming CI passed, LGTM

File: src/System.IO.UnmanagedMemoryStream/tests/ArrayHelpers.cs [hunk 1]
```diff
@@ -66,6 +64,14 @@ public bool Equals(T[] x, T[] y)
                 }
                 for (int i = 0; i < x.Length; i++)
                 {
+                    if (x[i] == null)
+                    {
+                        if (y[i] == null)
+                        {
+                            return true;
```
Comment (sharwell) [inline src/System.IO.UnmanagedMemoryStream/tests/ArrayHelpers.cs]: This return statement changes the semantics of the method. It should actually be `continue;`.

File: src/System.IO.UnmanagedMemoryStream/tests/ArrayHelpers.cs [hunk 2]
```diff
@@ -66,6 +64,14 @@ public bool Equals(T[] x, T[] y)
                 }
                 for (int i = 0; i < x.Length; i++)
                 {
+                    if (x[i] == null)
```
Comment (sharwell) [inline src/System.IO.UnmanagedMemoryStream/tests/ArrayHelpers.cs]: Why not just use `EqualityComparer<T>.Default` for this block?

File: src/System.IO.UnmanagedMemoryStream/tests/ArrayHelpers.cs (hunk 1 above)
Comment (KrzysztofCwalina) [inline src/System.IO.UnmanagedMemoryStream/tests/ArrayHelpers.cs]: Good catch. Thanks! I submitted a bug fix.

File: src/System.IO.UnmanagedMemoryStream/tests/ArrayHelpers.cs (hunk 2 above)
Comment (KrzysztofCwalina) [inline src/System.IO.UnmanagedMemoryStream/tests/ArrayHelpers.cs]: Yeah, now that the code got complex enough (after several review comments), I probably should :-). I will do it opportunistically with a next commit.

**Pull request merged**